packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertListing.tsx

```diff
@@ -0,0 +1,114 @@
+import { TableCell } from 'src/components/TableCell';
+import { TableHead } from 'src/components/TableHead';
+import { TableRow } from 'src/components/TableRow';
+import { TableRowError } from 'src/components/TableRowError/TableRowError';
+import { TableRowLoading } from 'src/components/TableRowLoading/TableRowLoading';
+import { TableSortCell } from 'src/components/TableSortCell';
+import { StyledPlaceholder } from 'src/features/StackScripts/StackScriptBase/StackScriptBase.styles';
+import { useAllAlertDefinitionsQuery } from 'src/queries/cloudpulse/alerts';
+
+import { AlertTableRow } from './AlertTableRow';
+
+export const AlertListing = () => {
+  // These are dummy order and handlers, will replace them in the next PR
+  const order = 'asc';
+  const handleOrderChange = () => {
+    return 'asc';
+  };
+  const { data: alerts, isError, isLoading } = useAllAlertDefinitionsQuery();
+  if (alerts?.length === 0) {
+    return (
+      <Grid item xs={12}>
+        <Paper>
+          <StyledPlaceholder
+            icon={AlertIcon}
+            subtitle="Start Monitoring your resources."
+            title=""
+          />
+        </Paper>
+      </Grid>
+    );
+  }
+  return (
+    <Grid marginTop={2}>
+      <Table colCount={7} size="small">
+        <TableHead>
+          <TableRow>
+            <TableSortCell
+              active={true}
+              direction={order}
+              handleClick={handleOrderChange}
+              label="alertName"
+            >
+              Alert Name
+            </TableSortCell>
+            <TableSortCell
+              handleClick={() => {
+                'asc';
+              }}
+              active={true}
+              direction={order}
+              label="serviceType"
+              size="small"
+            >
+              Service Type
+            </TableSortCell>
+            <TableSortCell
+              active={true}
+              direction={order}
+              handleClick={handleOrderChange}
+              label="severity"
+            >
+              Severity
+            </TableSortCell>
+            <TableSortCell
+              active={true}
+              direction={order}
+              handleClick={handleOrderChange}
+              label="status"
+            >
+              Status
+            </TableSortCell>
+            <TableSortCell
+              active={true}
+              direction={order}
+              handleClick={handleOrderChange}
+              label="lastModified"
+            >
+              Last Modified
+            </TableSortCell>
+            <TableSortCell
+              active={true}
+              direction={order}
+              handleClick={handleOrderChange}
+              label="createdBy"
+              size="small"
+            >
+              Created By
+            </TableSortCell>
+            <TableCell actionCell />
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {isError === true && (
```

Comment:
No need to put `=== true` for a boolean value

```suggestion
          {isError && (
```

Reply:
Understood, it is redundant. Will make the change.